bootstrap/src/setupProxy.js

```diff
@@ -13,6 +13,8 @@ module.exports = function(app) {
             headers: {
                 host: domain.replace(/https:\/\//, ''),
                 origin: null,
+                // This is essential for Tiger backends. To ensure 401 flies when not authenticated and using proxy
+                "X-Requested-With": "XMLHttpRequest",
```

Comment:
Is this safe for bear? Can we avoid mentionning tiger in a file that will be used in bear too?